src/frontend/src/components/createnewproject/SelectForm.tsx

```diff
@@ -80,7 +80,7 @@ const SelectForm = ({ flag, geojsonFile, customFormFile, setCustomFormFile }) =>
   };
   useEffect(() => {
     if (customFormFile && !customFileValidity) {
-      dispatch(ValidateCustomForm(`${import.meta.env.VITE_API_URL}/projects/validate-form`, customFormFile));
+      dispatch(ValidateCustomForm(`${import.meta.env.VITE_API_URL}/projects/validate-form?debug=true`, customFormFile));
```

Comment:
Minor change: this shouldn't use `debug=true` as it doesn't do any actual validation then (it's to return and inspect the form if there are errors).

I will merge anyway then update on the branch 👍